### PR TITLE
en: update Thanos

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

change aggregate-multiple-cluster-monitor-data.md

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/1215
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/aggregate-multiple-cluster-monitor-data.md [hunk 1]
```diff
@@ -11,45 +11,99 @@ This document describes how to aggregate the monitoring data of multiple TiDB cl
...
 {{< copyable "shell-regular" >}}
 
+```shell
+kubectl port-forward -n ${thanos_namespace} svc/thanos-query 9090
 ```
-kubectl -n ${namespace} apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/monitor-with-thanos/thanos-query.yaml
-```
 
-In Thanos Query, a Prometheus corresponds to a Store, which corresponds to a TidbMonitor. After deploying Thanos Query, you can provide a unified query interface for monitoring data through Thanos Query's API.
+If you want to access the Thanos Query panel using NodePort or LoadBalancer, refer to the following docs:
```
Comment (qiancai) [inline en/aggregate-multiple-cluster-monitor-data.md:68]:
```suggestion
If you want to access the Thanos Query panel using NodePort or LoadBalancer, refer to the following documents:
```

File: en/aggregate-multiple-cluster-monitor-data.md [hunk 2]
```diff
@@ -11,45 +11,99 @@ This document describes how to aggregate the monitoring data of multiple TiDB cl
...
-{{< copyable "shell-regular" >}}
+    A configuration example is as follows.
 
-```
-kubectl -n ${namespace} apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/monitor-with-thanos/tidb-monitor.yaml
-```
+    {{< copyable "shell-regular" >}}
+
+    ```
+    kubectl -n ${namespace} apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/monitor-with-thanos/tidb- monitor.yaml
```
Comment (better0332) [inline en/aggregate-multiple-cluster-monitor-data.md:25]:
```suggestion
    kubectl -n ${namespace} apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/monitor-with-thanos/tidb-monitor.yaml
```

File: en/aggregate-multiple-cluster-monitor-data.md [hunk 3]
```diff
@@ -11,45 +11,99 @@ This document describes how to aggregate the monitoring data of multiple TiDB cl
...
+       - --query.replica-label=rule_replica
+       - --store=<TidbMonitorName1>-prometheus.<TidbMonitorNs1>:10901
+       - --store=<TidbMonitorName2>-prometheus.<TidbMonitorNs2>:10901
 ```
 
 ## Configure archives and storage of Thanos Sidecar
 
+> **Note:**
+>
+> To ensure successful configuration, you must first create the S3 bucket. If you choose AWS S3, refer to [Create AWS S3 Bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html) and [AWS S3 Endpoint List](https://docs.aws.amazon.com/general/latest/gr/s3.html) for instructions.
```
Comment (TomShawn) [inline en/aggregate-multiple-cluster-monitor-data.md:105]:
```suggestion
> To ensure successful configuration, you must first create the S3 bucket. If you choose AWS S3, refer to [AWS documentation - Create AWS S3 Bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html) and [AWS documentation - AWS S3 Endpoint List](https://docs.aws.amazon.com/general/latest/gr/s3.html) for instructions.
```

**Review by TomShawn — COMMENTED** [review]
Minor comments~

**Comment by qiancai** [comment]
Thanks @TomShawn for the nice change suggestions. I've applied all of them. PTAL again.

File: en/aggregate-multiple-cluster-monitor-data.md [hunk 4]
```diff
@@ -9,47 +9,101 @@ This document describes how to aggregate the monitoring data of multiple TiDB cl
...
+
+1. Log in to Grafana. 
+2. In the left navigation bar, select `Configuration` > `Data Sources`. 
+3. Add or modify a DataSource in the Prometheus type. 
+4. Set the URL under HTTP to `http://thanos-query.${thanos_namespace}:9090`.
 
 ## Add or remove TidbMonitor
 
-If you need to add, update, or remove a monitor store from the Thanos Query, update the `--store` configuration of Thanos Query, and perform a rolling update to the Thanos Query component.
+In Thanos Query, a Prometheus corresponds to a monitor store and also corresponds to a TidbMonitor. If you need to add, update, or remove a monitor store from the Thanos Query, update the `--store` configuration of the Thanos Query component, and perform a rolling update to the Thanos Query component.
```
Comment (TomShawn) [inline en/aggregate-multiple-cluster-monitor-data.md:84]:
```suggestion
In Thanos Query, a Prometheus instance corresponds to a monitor store and also corresponds to a TidbMonitor. If you need to add, update, or remove a monitor store from the Thanos Query, update the `--store` configuration of the Thanos Query component, and perform a rolling update to the Thanos Query component.
```

**Review by TomShawn — APPROVED** [review]
Rest LGTM

**Review by better0332 — APPROVED** [review]
LGTM

**Pull request merged**